astropy/modeling/polynomial.py

```diff
@@ -1049,9 +1051,11 @@ class Chebyshev2D(OrthoPolynomialBase):
     """
     _separable = False
 
-    def __init__(self, x_degree, y_degree, x_domain=None, x_window=[-1, 1],
-                 y_domain=None, y_window=[-1, 1], n_models=None,
+    def __init__(self, x_degree, y_degree, x_domain=None, x_window=None,
```

Comment:
I take it all these similar changes are to avoid using a list as the default. But why not a tuple? Too many other code changes resulting?